feature-utils/poly-look/README.md

```diff
@@ -11,22 +21,41 @@ polyLook is the visual component library for all features and applications inclu
 
 ## State of polyLook
 
-polyLook aims to be a framework agnostic component library for polyPod features. At the moment, however, it mainly contains prototypes for a potential approach, based on LitElement and React elements.
+polyLook aims to be a framework agnostic component library for polyPod
+features. At the moment, however, it mainly contains prototypes for a potential
+approach, based on Lit and React elements.
```

Comment:
Well, the LitElement stuff are the prototypes for a potential approach - the React elements are what's actually used in features right now.

Reply:
Right. That should totally be rephrased, i guess...